encoder/vaapiencoder_h264.cpp

```diff
@@ -743,13 +743,21 @@ void VaapiEncoderH264::resetParams ()
     DEBUG("resetParams, ensureCodedBufferSize");
     ensureCodedBufferSize();
 
+    if (intraPeriod() == 0) {
+        ERROR("intra period must larger than 0");
+        assert(0);
```

Comment:
Could we silent change this to other value? We can assert on exception. But assert on user input value maybe not so good especially when it in core library. U.Arite have this concern too.https://github.com/01org/libyami/issues/439